webapp/src/js/controllers/contacts-edit.js

```diff
@@ -126,6 +126,10 @@ angular.module('inboxControllers').controller('ContactsEditCtrl',
       ctrl.setEnketoEditedStatus(true);
     };
 
+    var resetFormError = function() {
+      ctrl.setEnketoError(null);
```

Comment:
You're updating the error value every time a an input is changed, even if the value was not set before. This is very noisy:
![image](https://user-images.githubusercontent.com/35681649/64433056-68b05780-d0c6-11e9-8082-316f9c803d54.png)

Can we only update the state when `ctrl.enketoError` is truthy and we have an error to reset? You might need to add the `enketoError` mapping in the `mapStateToTarget` function.